### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by mjbrender**
Included clear contribution strategy and rearranged some information to make it a little easier to read.

### CODE REVIEW

**Comment by mjbrender** [comment]
Verified all links work at the top of README.

**Comment by mjbrender** [comment]
Hey @lukebakken, here's what I was talking about. Shoot me feedback on here or directly and I'll clean up the PR. Same goes to @broach!

**Comment by lukebakken** [comment]
Documentation for contributing should be a link to the wiki page, with that wiki page updated:

https://github.com/basho/riak-dotnet-client/wiki/Contributing

Let me know if you have issues with permissions to editing the wiki.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+##Contributing
```
Comment (lukebakken) [inline CONTRIBUTING.md]: Please remove this file and instead have a "Contributing" section in README.md that links here:

https://github.com/basho/riak-dotnet-client/wiki/Contributing

Then, merge this content with that page. This is to not have duplicate documentation. Thanks!

File: README.md [hunk 2]
```diff
@@ -1,25 +1,39 @@
 Riak .NET Client - a .NET client for Riak
 =========================================
 
-**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://riak.basho.com/), [Basho](http://www.basho.com/)'s fault-tolerante, distributed KV store.
-
+**Riak .NET Client** is a .NET library which makes it easy to communicate with [Riak](http://basho.com/riak/), an open source, distributed database that focuses on high availability, horizontal scalability, and *predictable*
+latency. Both Riak and this code is maintained by [Basho](http://www.basho.com/).
+
+1. [Installation](#installation)
```
Comment (lukebakken) [inline README.md]: As with "Contributing", this section should link to the wiki:

https://github.com/basho/riak-dotnet-client/wiki/Installation

and the contents here should be merged with / overwrite what is in the wiki.

File: CONTRIBUTING.md (hunk 1 above)
Comment (mjbrender) [inline CONTRIBUTING.md]: Hey @lukebakken - given the common approach of having a CONTRIBUTING.md file in a repo these days, I'd prefer to delete the page on the wiki and leave it in the repo for this one case. All other docs can of course go in the wiki. Would that work ?

File: README.md (hunk 2 above)
Comment (mjbrender) [inline README.md]: Updating now - thanks!

Comment (mjbrender) [inline README.md]: Wiki has been updated - https://github.com/basho/riak-dotnet-client/wiki/Installation - this section is removed from the PR.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lukebakken) [inline CONTRIBUTING.md]: Sigh OK. Can you update Contributing in the wiki to point back to this file?

Comment (mjbrender) [inline CONTRIBUTING.md]: Yessir. Thanks for compromising.. I think it will help people who contribute across multiple repos inside our orgs.

Comment (mjbrender) [inline CONTRIBUTING.md]: Updated - https://github.com/basho/riak-dotnet-client/wiki/Contributing

:+1:

**Comment by lukebakken** [comment]
:+1: nice bashoman hugs, haha

**Comment by mjbrender** [comment]
It's about the little touches :smiley:

**Comment by lukebakken** [comment]
@mjbrender - care to do the same here? :smile: 

https://github.com/basho/riak-nodejs-client
https://github.com/basho/riak-nodejs-client/wiki/Contributing

**Comment by mjbrender** [comment]
@lukebakken :+1:

**Pull request merged**